src/app_service/service/message/utils/link_preview_utils.nim

```diff
@@ -0,0 +1,80 @@
+import chronicles
+import ../dto/[link_preview, link_preview_thumbnail, status_community_link_preview, status_community_channel_link_preview]
+import ../../community/dto/community
+import ../../chat/dto/chat
+
+# This file is to avoid a recursive import of status_community_link_preview
+
+proc setCommunityInfo*(self: StatusCommunityLinkPreview, community: CommunityDto): bool =
+  if self.communityId != community.id:
+    return false
+
+  debug "setCommunityInfo", communityId = self.communityId, communityName = community.name
```

Comment:
Leftover?

Reply:
It came from the original function. Since it's a debug I don,t mind keeping it